llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldELF.cpp

```diff
@@ -1011,6 +1011,135 @@ void RuntimeDyldELF::resolveBPFRelocation(const SectionEntry &Section,
   }
 }
 
+static void applyUTypeImmRISCV(uint8_t *InstrAddr, uint32_t Imm) {
```

Comment:
I'm not sure if this was discussed elsewhere in the thread, but we should probably split these out into a separate PR

Reply:
Does this still need to be done?